serverless.yml

```diff
@@ -20,9 +20,10 @@ provider:
       - ${env:SUBNET_ID_B, 'subnetIdB'}
       - ${env:SUBNET_ID_C, 'subnetIdC'}
   environment:
-    RDF4J_SERVICE_URL: ${env:RDF4J_SERVICE_URL, cf:rdf4jLoadBalancerStack.RDF4JServiceUrl, 'http://localhost:8080'} 
+    RDF4J_SERVICE_URL: ${env:RDF4J_SERVICE_URL, cf:rdf4jLoadBalancerStack.RDF4JServiceUrl, 'http://localhost:8080'}
     RDF4J_USER_NAME: ${env:RDF4J_USER_NAME, 'rdf4j'}
     RDF4J_PASSWORD: ${env:RDF4J_PASSWORD, 'rdf4j'}
+    CMR_BASE_URL: ${env:CMR_BASE_URL, 'https://cmr.earthdata.nasa.gov'}
```

Comment:
how does this process support sit or uat?

Reply:
the 2. element is just for backup